tests/test_test_plugins.py

```diff
@@ -15,12 +15,12 @@
 
 
 class TestPluginTestRunner:
-	def setup_method(self):
-		DUMMY_PLUGIN_PATH.mkdir(parents=True, exist_ok=True)
-		Path(DUMMY_REQUIREMENTS).touch()
-		Path(DUMMY_TESTFILE).touch()
-		with open(DUMMY_TESTFILE, 'w') as f:
-			f.write(textwrap.dedent('''\
+    def setup_method(self):
+        DUMMY_PLUGIN_PATH.mkdir(parents=True, exist_ok=True)
+        Path(DUMMY_REQUIREMENTS).touch()
+        Path(DUMMY_TESTFILE).touch()
+        with open(DUMMY_TESTFILE, 'w') as f:
+            f.write(textwrap.dedent('''\
```

Comment:
fyi this is leading to errors : `TabError: inconsistent use of tabs and spaces in indentation`